girder/constants.py

```diff
@@ -195,6 +195,12 @@ class TokenScope:
     ANONYMOUS_SESSION = 'core.anonymous_session'
     USER_AUTH = 'core.user_auth'
     TEMPORARY_USER_AUTH = 'core.user_auth.temporary'
+    PLUGINS_ENABLED_READ = 'core.plugins.read'
```

Comment:
The burden of compatibility might not be quite so bad for these scopes. We shouldn't remove symbols under `TokenScope` once they're in master in case plugins are using them, but I think we should be OK with the idea of changing core and plugin endpoint scope requirements; the burden on users would simply be to log-out and reauthorize the app, which isn't so bad.